index.js

```diff
@@ -148,18 +148,25 @@ async function cloneExample() {
     await run(`git clone --depth 1 --filter=blob:none --sparse ${config.examples.repoUrl} ${tmpDir}`);
     // Checkout just the example dir.
     await run(`cd ${tmpDir} && git sparse-checkout set ${args.example}`);
-    // Copy out into a new directory within current working directory.
-    await run(`cp -R ${tmpDir}/${args.example} ${dirName}`);
+
+    // move out into a new directory.
+    await fs.moveSync(`${tmpDir}/${args.example}`, dirName);
```

Comment:
Why move and not copy?

Reply:
Move is always faster than having to copy one by one then having to delete it in memory. When you move, you remove it from memory at that point but for copy, you'll need to do a clean up. 

This is a performance improvement.